jsonrpc/src/schemas/trace.json

```diff
@@ -0,0 +1,95 @@
+{
+  "traceResultObject": {
+    "title": "traceResultObject",
+    "description": "Trace data for the result of tracing content request.",
+    "type": "object",
+    "required": [
+      "origin",
+      "targetId",
+      "responses",
+      "metadata",
+      "startedAtMs"
+    ],
+    "properties": {
+      "origin": {
+        "description": "Local Node ID",
+        "$ref": "#/components/schemas/uint256"
+      },
+      "targetId": {
+        "description": "Target content ID",
+        "$ref": "#/components/schemas/uint256"
+      },
+      "receivedFrom": {
+        "description": "Node ID from which the content was received.",
+        "$ref": "#/components/schemas/uint256"
+      },
+      "responses": {
+        "$ref": "#/components/schemas/traceResultResponses"
+      },
+      "metadata": {
+        "$ref": "#/components/schemas/traceResultMetadata"
+      },
+      "startedAtMs": {
+        "type": "integer",
+        "minimum": 0,
+        "description": "Timestamp of the beginning of this request in milliseconds."
+      },
+      "cancelled": {
+        "description": "List of node IDs requests to which were sent but cancelled due to receiving content from somewhere else, before receiving response from those nodes.",
+        "$ref": "#/components/schemas/listOfNodeIds"
+      }
+    }
+  },
+  "listOfNodeIds": {
+    "title": "listOfNodeIds",
+    "description": "Contains list of nodes IDs.",
+    "type": "array",
+    "items": {
+      "$ref": "#/components/schemas/uint256"
+    }
+  },
+  "traceResultResponses": {
+    "title": "traceResultResponses",
+    "description": "Contains a map of remote node IDs with the node IDs each node responded with. For the node ID that is in the `receivedFrom` field, `respondedWith` MUST be empty array.",
+    "type": "object",
+    "additionalProperties": {
+      "$ref": "#/components/schemas/traceResultResponseItem"
+    }
+  },
+  "traceResultResponseItem": {
+    "title": "traceResultResponseItem",
+    "description": "Contains the node's response, including the duration of the request.",
+    "type": "object",
+    "properties" : {
+      "durationsMs": {
+        "description": "Time it took from the beginning of the original request up to receiving this response.",
```

Comment:
Still have my original question unresolved: 
"Original request meaning the individual request or the start of the lookup, aka the json-rpc request?"